container/options.go

```diff
@@ -216,6 +245,23 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// SplitFixed sets the size as a fixed value of the available space.
```

Comment:
(nit) It is a bit unclear which size we are talking about. Can we say:

// SplitFixed sets the relative size of the split to a fixed number of cells.

Or similar.

Reply:
Went with:
```
// SplitFixed sets the size of the first container to be a fixed value
// and makes the second container take up the remaining space.
```
A bit longer, but a bit easier to understand imo. Is this ok or should I change it to
`// SplitFixed sets the relative size of the split to a fixed number of cells.`?